SparkPost/2web2ui
# SA-2213 - (UI) Add <ScreenReaderOnly /> tables that render Analytics Report chart data to the DOM

[SA-2213](https://sparkpost.atlassian.net/browse/SA-2213)

### What Changed

- Adds new `ChartScreenReaderOnly` table that renders alongside  each rendered chart
- Adjusts test fixtures for the sake of realism
- Adds supporting tests
- Removes the Codecov badge from our README - just happened to see it
- **NEW** Adds global `ScreenReaderOnlyTable` component with supporting tests and documentation.

### How To Test

- Ensure a `<table />` renders in the DOM for each chart, accurately rendering the data that the chart displays
- To make this easier to test, you can remove the `<ScreenReaderOnly />` component from `Charts.js` temporarily

### To Do

- [x] Hide tables with `<ScreenReaderOnly />`
- [x] Address unique table captions when active comparisons exist
- [x] Add tests for multiple comparison tables
- [x] Handle `industry_rate` data - talking with Patrick for API docs
